compute/gce/instance.go

```diff
@@ -0,0 +1,313 @@
+
+	token.SetAuthHeader(Startnoderequest)
+
+	Startnoderesp, err := client.Do(Startnoderequest)
+
+	defer Startnoderesp.Body.Close()
+
+	body, err := ioutil.ReadAll(Startnoderesp.Body)
+
+	fmt.Println(string(body))
+
+	return
+}
+
+//stop gce instance currentnly running
+//accept projectid, zone, instance
+
+func (gce *GCE) Stopnode(request interface{}) (resp interface{}, err error) {
+
+	options := request.(map[string]string)
+
+	url := "https://www.googleapis.com/compute/v1/projects/" + options["projectid"] + "/zones/" + options["Zone"] + "/instances/" + options["instance"] + "/stop"
+
+	token := sign()
+
+	client := &http.Client{}
+
+	Stopnoderequest, err := http.NewRequest("POST", url, nil)
+
+	Stopnoderequest.Header.Set("Content-Type", "application/json")
+
+	token.SetAuthHeader(Stopnoderequest)
+
+	Stopnoderesp, err := client.Do(Stopnoderequest)
+
+	defer Stopnoderesp.Body.Close()
+
+	body, err := ioutil.ReadAll(Stopnoderesp.Body)
+
+	fmt.Println(string(body))
+
+	return
+}
+
+//delete gce instance currentnly running
+//accept projectid, zone, instance
+
+func (gce *GCE) Deletenode(request interface{}) (resp interface{}, err error) {
+
+	options := request.(map[string]string)
+
+	url := "https://www.googleapis.com/compute/v1/projects/" + options["projectid"] + "/zones/" + options["Zone"] + "/instances/" + options["instance"]
+
+	token := sign()
+
+	client := &http.Client{}
+
+	Deletenoderequest, err := http.NewRequest("DELETE", url, nil)
+
+	Deletenoderequest.Header.Set("Content-Type", "application/json")
+
+	token.SetAuthHeader(Deletenoderequest)
+
+	Deletenoderesp, err := client.Do(Deletenoderequest)
+
+	defer Deletenoderesp.Body.Close()
+
+	body, err := ioutil.ReadAll(Deletenoderesp.Body)
+
+	fmt.Println(string(body))
+
+	return
+}
+
+//reboot/reset gce instance currentnly ***running***
+//accept projectid, zone, instance
+
+func (gce *GCE) Rebootnode(request interface{}) (resp interface{}, err error) {
+
+	options := request.(map[string]string)
+
+	url := "https://www.googleapis.com/compute/v1/projects/" + options["projectid"] + "/zones/" + options["Zone"] + "/instances/" + options["instance"] + "/reset"
+
+	fmt.Println(url)
+
+	token := sign()
+
+	client := &http.Client{}
+
+	Rebootnoderequest, err := http.NewRequest("POST", url, nil)
+
+	Rebootnoderequest.Header.Set("Content-Type", "application/json")
+
+	token.SetAuthHeader(Rebootnoderequest)
+
+	Rebootnoderesp, err := client.Do(Rebootnoderequest)
+
+	defer Rebootnoderesp.Body.Close()
+
+	body, err := ioutil.ReadAll(Rebootnoderesp.Body)
+
+	fmt.Println(string(body))
+
+	return
+}
+
+//list gce instance currentnly created
+//accept projectid, zone
+
+func (gce *GCE) listnode(request interface{}) (resp interface{}, err error) {
+
+	options := request.(map[string]string)
+
+	url := "https://www.googleapis.com/compute/v1/projects/" + options["projectid"] + "/zones/" + options["Zone"] + "/instances/"
+
+	fmt.Println(url)
+
+	token := sign()
+
+	client := &http.Client{}
+
```

Comment:
Lot of unnecessary line breaks. Use line breaks only when you need to group out set of function calls.